ui/components/multichain/app-header/app-header-unlocked-content.tsx

```diff
@@ -244,7 +251,7 @@ export const AppHeaderUnlockedContent = ({
                   if (!isEvmNetwork) {
                     return;
                   }
```

Comment:
This shouldn't be disabled here

Reply:
This is more for BTC accounts, since we aren't it to connect to other sites. For evm accounts, the connected status indicator would work like normal